src/Microsoft.PowerShell.Commands.Management/commands/management/CombinePathCommand.cs

```diff
@@ -29,10 +30,11 @@ public class JoinPathCommand : CoreCommandWithCredentialsBase
         /// <summary>
         /// Gets or sets the childPath parameter to the command
         /// </summary>
-        [Parameter(Position = 1, Mandatory = true, ValueFromPipelineByPropertyName = true)]
+        [Parameter(Position = 1, Mandatory = true, ValueFromPipelineByPropertyName = true, ValueFromRemainingArguments = true)]
         [AllowNull]
         [AllowEmptyString]
-        public string ChildPath { get; set; } = String.Empty;
+        [AllowEmptyCollection]
+        public string[] ChildPath { get; set; } = new string[0];
```

Comment:
I figured that PowerShell will happily bind a string to a string[] parameter anyway, so no harm done. :)  (Unless someone's using the cmdlet directly in C#,but that seems unlikely.)

> On Aug 22, 2016, at 11:16 AM, Jason Shirk <notifications@github.com> wrote:
> 
> In src/Microsoft.PowerShell.Commands.Management/commands/management/CombinePathCommand.cs:
> 
> >          [AllowNull]
> >          [AllowEmptyString]
> > -        public string ChildPath { get; set; } = String.Empty;
> > +        [AllowEmptyCollection]
> > +        public string[] ChildPath { get; set; } = new string[0];
> Note that changing the type on a public API is normally considered a breaking change.
> 
> That said, cmdlets are too hard to use as an API, so it's unlikely to break anybody.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>